my_init.d/factory_reset.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+# Debugging.
+set -x
+
+# Environment.
+export SHELL=/bin/bash
+
+# Performing the factory reset of your AiiDAlab environment:
+# 0 - do nothing
+# 1 - remove aiidalab apps and things installed in the .local folder
+# 2 - remove the entire content of the home folder.
+
+if [[ "${AIIDALAB_FACTORY_RESET}" == 0 ]]; then
+  exit 0
+fi
+
+if [[ "${AIIDALAB_FACTORY_RESET}" == 1 ]]; then
+  rm -r /home/${SYSTEM_USER}/apps
+  rm -r /home/${SYSTEM_USER}/.local
+
+elif [[ "${AIIDALAB_FACTORY_RESET}" == 2 ]]; then
+  rm -r /home/${SYSTEM_USER}/*
+
+fi
```

Comment:
This would be a prime use case for [case statements](https://linuxize.com/post/bash-case-statement/).

But this is only relevant if we don't adopt the bit mask approach I recommended in an earlier comment.

Reply:
We discussed elsewhere and agreed to take the `case statements` approach for the moment. In case of a need, we can switch to the mask approach later.